prac_06/guitars.py

```diff
@@ -0,0 +1,44 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    print("My Guitars!")
+
+    guitars = []
+    guitar_number = 1
+
+    name = input("Name: ")
+
+    while name != "":
+        try:
+            year = int(input("Year: "))
+            cost = float(input("Cost: $"))
+        except ValueError:
+            print("Invalid input. Please enter a valid year and cost.")
+            name = input("Name: ")
+            continue
+
+        if year < 0 or cost < 0:
+            print("Year and cost must be non-negative.")
+            name = input("Name: ")
+            continue
```

Comment:
You could also create a reusable function by passing the list and a prompt as parameters